Makefile

```diff
@@ -199,3 +167,25 @@ adr-gen:
 	@echo "--> Downloading ADR template"
 	@curl -sSL https://raw.githubusercontent.com/celestiaorg/.github/main/adr-template.md > docs/architecture/adr-template.md
 .PHONY: adr-gen
+
+## goreleaser: list goreleaser options and checks if goreleaser is installed
+goreleaser: Makefile
```

Comment:
[question] Is the `Makefile` command a prerequisite of `goreleaser`? I don't think a `Makefile` command exists.

```suggestion
goreleaser:
```

Reply:
This passes the Makefile in for the `sed` command to parse. This is how the `help` command works.

Reply:
Thanks! Resolving.